.cirrus.yml

```diff
@@ -132,3 +140,20 @@ promote_task:
     folder: ${CIRRUS_WORKING_DIR}/.m2/repository
   script: cirrus_promote_maven
   cleanup_before_cache_script: cleanup_maven_repository
+
+build_and_release_dotnet_task:
```

Comment:
Can we rename the task? It seems a bit confusing to me as it does not do the release and if I am not mistaken it will not do it in the future either. As the whole dotnet logic will be part of this task it should somehow reflect that. And at the same time I don't have a really good suggestion how to name it.

Reply:
I can rename it to just `build` but this is a scaffold of the pipeline and it will change again anyways. 
So we might leave it as is for this PR? Unless there's already a plan of how we want to organize the tasks/scripts.

Reply:
ok, leave it as is and we will rename when adding the other steps.